### PR TITLE
Test fixes for planemo machine

### PR DESCRIPTION
**Pull request by jmchilton**
First round of fixes for running ``bioblend-galaxy-tests`` against planemo-machine. See individual commits for individual descriptions.

### CODE REVIEW

File: bioblend/_tests/test_util.py [hunk 1]
```diff
@@ -60,8 +60,17 @@ def skip_unless_galaxy(min_release=None):
...
+            try:
+                config = gi.config.get_config()
+            except Exception:
+                # If older Galaxy for instance just assume use_remote_user is False.
+                config = {}
+
+            if config.get("use_remote_user", False):
+                new_user = gi.users.create_remote_user(galaxy_user_email)
+            else:
+                # Create a new user and get a new API key for her
```
Comment (nsoranzo) [inline bioblend/_tests/test_util.py]: You can move `galaxy_user` and `galaxy_password` definitions (lines 60-61) here.

Comment (jmchilton) [inline bioblend/_tests/test_util.py]: Yup - that makes a lot of sense. I've rebased that last commit with that change. Thanks a bunch 🦅👀 !

**Comment by nsoranzo** [comment]
Thanks @jmchilton!

**Comment by jmchilton** [comment]
Thanks for the merge @nsoranzo!

**Pull request merged**